app/code/Magento/InventoryInStorePickupAdminUi/view/adminhtml/templates/order/create/storepickup-script-load.phtml

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
+
+<script type="text/x-magento-init">
+    {
+        "*": {
+            "Magento_Ui/js/core/app": {
+                "components": {
+                    "triggerShippingMethodUpdate": {
+                        "component": "Magento_InventoryInStorePickupAdminUi/order/create/trigger-shipping-method-update"
+                    }
+                }
+            }
+        }
+    }
+</script>
```

Comment:
Missed empty line in the end of the file.

Reply:
Fixed